content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -33,7 +167,7 @@ Follow these steps to deploy Kubeflow:
 # Add kfctl to PATH, to make the kfctl binary easier to use.
 export PATH=$PATH:"<path to kfctl>"
 export KFAPP="<your choice of application directory name>"
-export CONFIG="https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_existing_arrikto.0.6.yaml"
+export CONFIG="https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_existing_arrikto.yaml"
```

Comment:
The link to the YAML file gives a 404.

Reply:
The name of the file is changed in this PR: https://github.com/kubeflow/kubeflow/pull/3857
Once it goes through, this comment should be addressed.

Reply:
Thanks @yanniszark 
@sarahmaddox I should have mentioned this